### PR TITLE
Integrate new Avail-SubXt version

### PR DESCRIPTION
**Pull request by fmiguelgarcia**
- Use the new `avail-subxt` available on `miguel/new_chainspec` node repo.
- Remove `subxt` dependency, because needed symbols are now re-exported in `avail-subxt`

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,9 +7,8 @@ authors = ["Polygon Avail Team"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = { git = "https://github.com/availproject/avail-core.git", tag = "kate-recovery/v0.8.1" }
-avail-subxt = { git = "https://github.com/availproject/avail.git", tag = "v1.5.0" }
-subxt = "0.27.1"
+kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
+avail-subxt = { path = "../avail_1/avail-subxt" }
```
Comment (aterentic-ethernal) [inline Cargo.toml:11]: Is this intentional?

File: src/network/client.rs [hunk 2]
```diff
@@ -331,6 +331,7 @@ impl Client {
 	}
 }
 
+#[allow(dead_code)]
```
Comment (aterentic-ethernal) [inline src/network/client.rs:334]: Lets leave warnings, we will remove dead code in separate PR.

File: Cargo.toml (hunk 1 above)
Comment (fmiguelgarcia) [reply Cargo.toml]: Yes, I was using this meanwhile the `avail-core` PR was in progress. 
Now it is fixed and pointing to a proper tagged version.

File: Cargo.toml [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 authors = ["Polygon Avail Team"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = { git = "https://github.com/availproject/avail-core.git", tag = "kate-recovery/v0.8.1" }
-avail-subxt = { git = "https://github.com/availproject/avail.git", tag = "v1.5.0" }
-subxt = "0.27.1"
+kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
+avail-subxt = { git = "https://github.com/availproject/avail.git", tag = "v1.6.0" }
```
Comment (aterentic-ethernal) [inline Cargo.toml:11]: We have to update runtime node version to 1.6.0 in `main.rs` also.

**Review by kroos47 — APPROVED** [review]

**Review by sh3ll3x3c — APPROVED** [review]

**Pull request merged**